packages/@okta/vuepress-site/docs/guides/submit-app/main/index.md

```diff
@@ -60,13 +60,48 @@ If the Okta OIN team identifies any issues in the initial review and QA testing
 
   >**Note:** SWA app integrations are no longer accepted for publication in the OIN catalog. However, existing SWA apps are still maintained by the OIN team.
 
+### Understand the submission review process
+
+The submission review process begins after you click **Submit for Review** in the [OIN Manager](https://oinmanager.okta.com). Okta sends you an email notification that your integration is now queued for review by the Okta OIN team, along with a date by which you should complete the initial review.
```

Comment:
maybe: "...begins after you click..." > "...begins when you click..." ?

Would it be inferred that the reader will be the recipient of the notification? In that case we could simplify to:
"Okta sends an email notification that your integration..."

Also, I don't think that the reader needs to complete the initial review? Isn't the OIN team doing the review? If so, maybe something like:
"...the Okta OIN team, which includes the date that the initial review of the integration is expected to be complete."

Reply:
Thanks Tim. I'll implement all of your suggestions, except the "Okta sends you an email notification" (I wanted to be clear that the recipient of the notification is the person who submitted the integration).